<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderPassEditor</class>
 <widget class="QWidget" name="ShaderPassEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="audio_message">
     <property name="text">
      <string>The audio data will be automatically passed to this shader. Make sure to enabled &quot;Use audio&quot; in the effect page.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCodeEditor" name="code" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="texture_preview">
     <property name="maximumSize">
      <size>
       <width>512</width>
       <height>512</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="choose_texture">
     <property name="text">
      <string>Choose texture</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCodeEditor</class>
   <extends>QWidget</extends>
   <header location="global">Dependencies/QCodeEditor/include/internal/QCodeEditor.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
